includes/multi-currency/Analytics.php

```diff
@@ -263,10 +264,10 @@ public function filter_select_clauses( array $clauses, $context ): array {
 		}
 
 		if ( $this->is_supported_context( $context ) && ( in_array( $context_page, self::SUPPORTED_CONTEXTS, true ) || $this->is_order_stats_table_used_in_clauses( $clauses ) ) ) {
-			$new_clauses[] = ', wcpay_multicurrency_currency_postmeta.meta_value AS order_currency';
-			$new_clauses[] = ', wcpay_multicurrency_default_currency_postmeta.meta_value AS order_default_currency';
-			$new_clauses[] = ', wcpay_multicurrency_exchange_rate_postmeta.meta_value AS exchange_rate';
-			$new_clauses[] = ', wcpay_multicurrency_stripe_exchange_rate_postmeta.meta_value AS stripe_exchange_rate';
+			$new_clauses[] = ', wcpay_multicurrency_currency_meta.meta_value AS order_currency';
```

Comment:
Will this vary based on the COT check as well like we did the `meta_table` below?

Reply:
Unless I've misunderstood the code, I don't think it would need to, because these are just alises we assign to the tables in `filter_join_clauses`. These would be the table aliases regardless of whether COT is enabled or not.

Reply:
Exactly. All meta tables in WordPress follow the same structure, the only difference being their names and the name of the foreign key column (ex. `post_id`, `comment_id`, `order_id`, etc.), so that they can be used with [`WP_Meta_Query`](https://developer.wordpress.org/reference/classes/wp_meta_query/) afterwards.

In this class we're joining the tables based on the foreign key column (`post_id` or `order_id`), and giving them aliases within `filter_join_clauses`. Afterwards all other queries utilize the same set of fields (`meta_key` and `meta_value`), so there is no need for changes there.